contracts/mocks/DCAStrategies/DCAStrategiesPermissionsHandlerMock.sol

```diff
@@ -5,6 +5,7 @@ import '../../DCAStrategies/DCAStrategies/DCAStrategiesPermissionsHandler.sol';
 
 contract DCAStrategiesPermissionsHandlerMock is DCAStrategiesPermissionsHandler {
   uint256 private _blockNumber;
+  mapping(uint256 => IDCAStrategies.Permission[]) private _setPermissionsCalls;
```

Comment:
By doing this, you are only recording once per token id. It could happen that you are calling the same token id multiple times and you wouldn't realize

I suggest you create a struct of {tokenId, IDCAStrategies.Permission[]} like [this](https://github.com/Mean-Finance/dca-v2-core/blob/main/contracts/mocks/DCAPermissionsManager/DCAPermissionsManager.sol#L8)

Reply:
I had problems with doing it that way but I will find a workaround